internal_user/templates/issueCheque.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags%}
+{% block javascript %}
+ <script>
+    $(function(){
+         $('#id_chequeAmount').keyup(function(e){
+            var amount = $("#id_chequeAmount")[0].value;
+            if(amount == ""){
+                $('#issue_cheque').prop("disabled", true);
+                $('#request_cheque').prop("disabled", true);
+            }
+            else if(amount>500){
```

Comment:
Is this 500 mentioned somewhere in the requirements? Don't remember.

Reply:
500 is the threshold which defines if a transaction is critical or not. I know it's not good practice to hard code values, I'm searching how to save all constants in one file, in the meantime you guys can use it to develop bachend